src/app/app.component.ts

```diff
@@ -7,4 +8,10 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'dashboard-ci';
+  public vendor: any = false;
+  constructor(public translateService: TranslateService) {
+    this.translateService.getByFileName('VENDOR', 'default-en').subscribe(value => {
```

Comment:
Can we have buttons in the UI for language selection and set the language based on that?

Reply:
Current there is only one language file! so we can't give button now, later if we increase the language file then we can put the button!!